lighthouse-core/test/gather/gatherers/dobetterweb/response-compression-test.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+const ResponseCompression =
+    require('../../../../gather/gatherers/dobetterweb/response-compression');
+const assert = require('assert');
+
+let options;
+let optimizedResponses;
+const traceData = {
+  networkRecords: [
+    {
+      _url: 'http://google.com/index.js',
```

Comment:
@paulirish yeah I've run into this a number of times now too, at first I was following the pattern of the private variables, but have mixed in the public accessors in later audits. Are we trying to standardize on the public ones now?